src/components/Button/FavoriteBox.tsx

```diff
@@ -27,12 +27,12 @@ export default React.memo(function FavoriteBox(props: FavoriteBoxProps) {
     >
       <Icon
         name="heart outline"
-        className={active ? "button-box__icon" : "button-box__icon-hovered"}
+        className={!active ? "button-box__icon" : "button-box__icon-hovered"}
```

Comment:
If we follow BEM, in this case, `active` is a state of the button which makes the CSS as follow 

```suggestion
        className={TokinList.join([
          "button-box__icon",
          !!active && "button-box__icon--active"
        ])}
```